### PR TITLE
ci: match gh action job names with mergifyio rules

### PR DESCRIPTION
**Pull request by Rakshith-R**
Refer: https://docs.mergify.com/conditions/#github-actions

Signed-off-by: Rakshith R <rar@redhat.com>

check summary here 
https://github.com/csi-addons/kubernetes-csi-addons/pull/25
https://github.com/csi-addons/kubernetes-csi-addons/pull/25/checks?check_run_id=4605948550

### CODE REVIEW

**Review by nixpanic — DISMISSED** [review]

**Review by Madhu-1 — DISMISSED** [review]

**Review by yati1998 — APPROVED** [review]

File: .github/mergify.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ pull_request_rules:
       - "#approved-reviews-by>=2"
       - "#changes-requested-reviews-by=0"
       - "status-success=codespell"
-      - "status-success=build_container"
+      - "status-success=build_controller"
```
Comment (Rakshith-R) [inline .github/mergify.yml:25]: This needed change too, :laughing: container -> controller

**Comment by nixpanic** [comment]
Can you please put this in a single commit?

**Comment by Rakshith-R** [comment]
> Can you please put this in a single commit?

done

**Review by nixpanic — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Comment by nixpanic** [comment]
Needs manual merging, and does not affect building the container images.

**Pull request merged**